schemas/src/elements/inputs/Input.json

```diff
@@ -10,6 +10,16 @@
 			"description": "Unique identifier for the value. Used to identify collected input when the Submit action is performed.",
 			"override": true
 		},
+		"errorMessage": {
+			"type": "string",
+			"description": "Error message to display when entered input is invalid",
+			"version": "1.3"
+		},
+		"isRequired": {
+			"type": "boolean",
+			"description": "Whether or not this input is required",
+			"version": "1.3"
+		},
 		"label": {
 			"type": "string|TextBlock|RichTextBlock",
```

Comment:
>|TextBlock|RichTextBlock [](start = 18, length = 24)

If we're not supporting this for v1 we should remove these from here. In general we should make sure the schema files are up-to-date with the latest spec changes.